docs/shared/packages/storybook/interaction-tests.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Setting up Storybook interaction tests with Nx
+description: This guide explains how you can set up Storybook interaction tests on your Nx workspace.
+---
+
+# Setting up Storybook interaction tests with Nx
```

Comment:
Just noticed I commented on the generated MD file, so plz have a look at my comments above 😅